src/content/app/genome-browser/state/browser-nav/browserNavSlice.ts

```diff
@@ -25,10 +24,10 @@ export enum BrowserNavAction {
 }
 
 export const browserNavButtonActionMap = {
-  [BrowserNavAction.MOVE_LEFT]: OutgoingActionType.MOVE_LEFT,
-  [BrowserNavAction.MOVE_RIGHT]: OutgoingActionType.MOVE_RIGHT,
-  [BrowserNavAction.ZOOM_OUT]: OutgoingActionType.ZOOM_OUT,
-  [BrowserNavAction.ZOOM_IN]: OutgoingActionType.ZOOM_IN
+  // [BrowserNavAction.MOVE_LEFT]: OutgoingActionType.MOVE_LEFT,
```

Comment:
Do you still want to keep this? There is no more `OutgoingActionType`

Reply:
Good point. We aren't using `browserNavSlice` at all anymore. I deleted the slice file and the selector file. Should have deleted them back in https://github.com/Ensembl/ensembl-client/pull/914.